features/FEATURE_BLE/ble/ArrayView.h

```diff
@@ -83,17 +83,16 @@ struct ArrayView {
      * a pointer to elements.
      */
     template<size_t Size>
-    ArrayView(T (&elements)[Size]):
-		_array(elements), _size(Size) { }
+    ArrayView(T(&elements)[Size]):
```

Comment:
Where is it mentioned that we should remove space in reference to array declaration ?

Reply:
I'll reply here, there are few similar comments. From the documentation:

> Artistic Style cannot always determine the usage of symbols with more than one meaning. For example an asterisk (*) can be multiplication, a pointer, or a pointer dereference. The "&" and "&&" symbols are a similar problem.
> If a symbol is being padded incorrectly, padding it manually may fix the problem. If it is still being padded incorrectly, then disabling the formatting may be necessary. To avoid having to use the "disable block" tags above, a single line disable is available.

😞 

I'll play with options around padding. I fear we leave padding out of it (3 pad rules that are currently defined).

Reply:
IME, AStyle gets it right 99% of the time, at least for normal C code. This sort of complex C++ templatery is probably pushing it too far though.

I'd be fine with having the pad rules as long as errors could be cleaned up manually and not cause a total CI blockage if AStyle sees something it disagrees with in future. (Can you manage to make the CI only check areas touched by diffs, and not recomplain about existing code?)

Reply:
@kjbracey-arm That would be best approach (using git to get list of files that are changed in the PR). 
I would like to address some comments here, but the best way to go forward would be to find a best match of astyle that works on our code base C/C++ and use it for all new files (using git to return what files are changed).  We can do cleanup of the codebase even separately